addons/hw_drivers/tools/helpers.py

```diff
@@ -136,48 +136,44 @@ def check_certificate():
         return {"status": CertificateStatus.OK, "message": message}
 
 
-def check_git_branch():
-    """
-    Check if the local branch is the same than the connected Odoo DB and
+def check_git_branch(force_checkout=False):
+    """Check if the local branch is the same as the connected Odoo DB and
     checkout to match it if needed.
+
+    :param force_checkout: Force the checkout to the DB branch
     """
     server = get_odoo_server_url()
```

Comment:
```suggestion
    server = get_odoo_server_url()
    if not server:
        return
```

Since the CRON job now runs every Monday even if we're not connected to a DB, we need a check here to that we don't get an error

Reply:
good point, thanks!